src/mount/sauna_client.h

```diff
@@ -79,8 +79,10 @@ struct FsInitParams {
 	static constexpr float    kDefaultBandwidthOveruse = 1.0;
 	static constexpr unsigned kDefaultChunkserverWriteTo = 5000;
 #ifdef _WIN32
+	static constexpr bool     kDefaultSpecialCopyMode = false;
 	static constexpr unsigned kDefaultWriteCacheSize = 50;
 #else
+	static constexpr bool     kDefaultIgnoreFlush = false;
```

Comment:
Consider using the same variable name for Unix and Windows platforms as they are intended for the same purpose and have identical initialization values.

Reply:
Well, they are used for the equivalent options: an option which tries to fool file-by-file copiers into believing file are fast copied to make them send the request to write another file, and we could process those writes in parallel.

But the option worked differently for both OSs, for the Linux only had to mock the flush syscall, but under Windows had to mock also the read. So, I'll leave it this way because I don't want to mix both.